packages/client/lib/components/ReportForm/FormViews/LocationView.jsx

```diff
@@ -0,0 +1,44 @@
+import useFormStore from '@/store/formStore';
+import { MapPinIcon } from '@heroicons/react/24/outline';
+
+const LocationView = () => {
+  const location = useFormStore((state) => state.location);
+  const setLocation = useFormStore((state) => state.setLocation);
+  const setStep = useFormStore((state) => state.setStep);
+
+
+  return (
+    <div className='flex-1'>
+      <h2 className="text-xl mt-4">Pin map location</h2>
+      <div>
+        <label htmlFor="location" className="block mb-2 text-black">
+          e.g. "Example Road, City" or "-8.54, 115.24"
+        </label>
+        <input
+          type="text"
+          id="location"
+          disabled
+          onChange={(e) => setLocation(e.target.value)}
+          name="location"
+          value={`${location.lat}, ${location.lng}`}
+          placeholder="Provide a location or coordinates"
+          className="block w-full p-2 text-black border border-gray-300 rounded-lg bg-white focus:ring-primary focus:border-primary shadow-md"
```

Comment:
```suggestion
          className="block w-full p-2 text-foreground border border-gray-300 rounded-lg bg-background focus:ring-primary focus:border-primary shadow-md"
```

Not sure about the borders off the top of my head but can you match to the figma spec? It keeps refactoring in the future A LOT easier if we want to update colors and there are only 5-6 of them and not like 40 +.

Reply:
This will have to be a work in progress. I changed all the white and black to foreground and background but I'll open a new issue to refactor all this.